### PR TITLE
Clean up issues related to connectedness and small (<=3) cardinalities.

### LINKED ISSUE #470: Careful handling of "Has a Dispersion Point" and "Biconnected" in spaces of cardinality less than four
"Has a Dispersion Point" (https://topology.pi-base.org/properties/P000045) says that a dispersion point is one whose removal makes a connected space totally disconnected. But this appears to mean that any connected space of size 1 or 2 has a dispersion point, since any space of size 0 or 1 is totally disconnected (https://topology.pi-base.org/properties/P000047). This contradicts, for example, "Indiscrete Topology on a Two-Point Set" (https://topology.pi-base.org/spaces/S000004/properties).

Maybe we should follow Wikipedia (https://en.wikipedia.org/wiki/Dispersion_point) and say that "Has a Dispersion Point" also requires the space to have at least 3 points.

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #470

### CODE REVIEW

**Comment by StevenClontz** [comment]
Despite my commit messages, this only captures 7/9 of the three-point topological spaces, missing 2&3 of https://en.wikipedia.org/wiki/Finite_topological_space#3_points

But it's unclear to me what properties we track that those would distinguish anyway, so this is probably more than enough, and apologies to the reviewer.

**Comment by StevenClontz** [comment]
Also, the motivation for this (biconnected/dispersion properties) isn't all there: we don't compute biconnected for S164, and don't compute dispersion point for three of the seven. But I need to put a lid on this for one PR.

File: theorems/T000092.md [hunk 1]
```diff
@@ -7,8 +7,10 @@ then:
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
+- mathse: 4812062
+  name: Relationship between dispersion point and biconnected property
 ---
 
 Suppose $p \in X$ is a dispersion point and $X = U \cup V$ is a disjoint union then without loss of generality $p \in U$ and $V \subset X \setminus \{p\}$ is disconnected.
```
Comment (prabau) [inline theorems/T000092.md:14]: Note that "disconnected" was never defined as such, so this is not clear and if it's means "not connected", it's even false when $V$ is a singleton or the empty set for example.  The sentence as written needs some rewording.  Maybe look at the proof in the mathse question for inspiration.

**Comment by prabau** [comment]
Added T41 [has dispersion point ==> not empty].  Plus some fixes for the theorems.

File: theorems/T000092.md (hunk 1 above)
Comment (prabau) [reply theorems/T000092.md]: fixed

**Review by StevenClontz — COMMENTED** [review]
bf9d0c16f1a685572c6bc2202a3a8edcd00e067f looks good to me

File: spaces/S000190/properties/P000001.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+value: false
+refs:
+- doi: 10.1007/978-1-4612-6290-9
+  name: Counterexamples in Topology
+---
+
+$X$ is not $T_0$ because the only open set containing
+distinct points $a, b \in X$ is $X$ itself, and thus,
+there exists no open set containing $a$ but not $b$ or
+containing $b$ but not $a$.
```
Comment (prabau) [inline spaces/S000190/properties/P000001.md:13]: The explanation (cut and pasted from S4) does not make full sense as written (and did not make full sense for S4 either). The first part says that the only open set containing two distinct points in the whole set.  But this proves nothing for the $T_0$ property.  The second part is ok.  How about drastically simplifying all that and only give a hint of the reason (maybe the fact that the only nonempty open set is $X$ itself), or just referring to Counterexamples?

**Comment by prabau** [comment]
Also, add a trait for Connected = false for S164.

File: spaces/S000010/README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ name: Sierpinski Space
 aliases:
   - Particular Point Topology on a Two-Point Set
   - Excluded Point Topology on a Two-Point Set
+  - Right-ordered Topology on a Two-Point Set
```
[the remark this answers is not in the document]
Comment (StevenClontz) [reply spaces/S000010/README.md:7]: Pushing a commit to standardize "Ray Topology" in our names (could use "right-ray" or "left-ray" for orders not isomorphic to their reversal).

File: spaces/S000190/properties/P000001.md (hunk 2 above)
Comment (StevenClontz) [reply spaces/S000190/properties/P000001.md]: I did one better - $T_0$ can be disproven by the indiscrete property so I just dropped the assertion and let the 🤖 do the work.

**Comment by prabau** [comment]
There a few previous comments that were left unresolved for S188: README, P1 and P2.

Not entirely sure about the terminology "Ray topology" by itself.  Is that common usage? In particular for S42 (Right-ordered reals) now called just "ray topology on the reals" (and the previous name "right-ordered reals" is gone). Left-ray and right-ray topologies are homeomorphic in this case, but "ray topology" makes it seem that it could be the topology generated by the various rays, both left rays and right rays, which ends up being the same as the usual order topology.  If we need to specify one side only, "left ray topology" and "right ray topology" seem better for definiteness.  Something to think about.

**Comment by StevenClontz** [comment]
In [6554448](https://github.com/pi-base/data/pull/471/commits/6554448dac76371f8265213f0a91f2d3068559bb) I made sure to add back "Right-ordered reals" as an alias (helpful to not break old searches if nothing else).

Looking around, I think "right ray topology" and "left ray topology" are both common, and more descriptive than "right-order" or "left-order". I somewhat like omitting left/right when the choice is irrelevant, where "ray topology" invokes "open sets are rays" rather than "all rays are open sets". 

But upon some reflection, perhaps we should standardize a choice of "right ray" across pi-Base where $\{y:x<y\}$ describes non-empty open sets, with "left ray" as an alias when they are equivalent. I think we want these spaces to appear when people search for "right ray" or "left ray" since they appear both ways, so both need to appear as aliases if not in the actual name. I'll make that contribution now.

**Comment by prabau** [comment]
Looks good.  Approving.

**Pull request merged**